src/lib/presentation/pages/album_details_page.dart

```diff
@@ -58,10 +66,98 @@ class _AlbumDetailsPageState extends State<AlbumDetailsPage> {
+                          return IconButton(
+                            key: GlobalKey(),
+                            icon: const Icon(Icons.more_vert_rounded),
+                            onPressed: () => _openMultiselectMenu(context),
+                          );
+
+                        return Container();
+                      },
+                    ),
+                    Observer(
+                      builder: (context) {
+                        final isMultiSelectEnabled = store.selection.isMultiSelectEnabled;
+                        final isAllSelected = store.selection.isAllSelected;
+
+                        if (isMultiSelectEnabled)
+                          return IconButton(
+                            key: GlobalKey(),
+                            icon: isAllSelected
+                                ? const Icon(Icons.deselect_rounded)
+                                : const Icon(Icons.select_all_rounded),
+                            onPressed: () {
+                              if (isAllSelected)
+                                store.selection.deselectAll();
+                              else
+                                store.selection.selectAll();
+                            },
+                          );
+
+                        return Container();
+                      },
+                    ),
+                    Observer(builder: (context) {
+                      final isMultiSelectEnabled = store.selection.isMultiSelectEnabled;
+                      return IconButton(
+                        key: const ValueKey('ALBUM_MULTISELECT'),
+                        icon: isMultiSelectEnabled
+                            ? const Icon(Icons.close_rounded)
+                            : const Icon(Icons.checklist_rtl_rounded),
+                        onPressed: () => store.selection.toggleMultiSelect(),
+                      );
+                    })
+                  ],
+                  cover: Container(
+                    clipBehavior: Clip.antiAlias,
+                    decoration: BoxDecoration(
+                      borderRadius: BorderRadius.circular(4.0),
+                      boxShadow: const [
+                        BoxShadow(color: Colors.black54, blurRadius: 8, offset: Offset(0, 2)),
+                      ],
+                      image: DecorationImage(
+                        image: utils.getAlbumImage(album.albumArtPath),
+                        fit: BoxFit.contain,
+                      ),
+                    ),
+                  ),
+                  background: ClipRect(
+                    clipBehavior: Clip.hardEdge,
+                    child: ImageFiltered(
+                      imageFilter: ImageFilter.blur(sigmaX: 20.0, sigmaY: 20.0),
+                      child: Image(
+                        image: utils.getAlbumImage(album.albumArtPath),
+                        fit: BoxFit.cover,
+                        opacity: const AlwaysStoppedAnimation(1.0),
```

Comment:
There's some function calculating this value in the old AlbumSliverAppBar. I decided to drop it since I didn't see much of a difference. If it has some use that I missed I can add it back.

Reply:
You're right. I don't see a difference either. We can even omit this parameter altogether.